packages/services-common/src/squidex/client.ts

```diff
@@ -0,0 +1,57 @@
+import Got from 'got';
+import decode from 'jwt-decode';
+import { cms as squidex } from '../config';
+
+interface JwtToken {
+  exp: number;
+}
+
+export default function create(): typeof Got {
+  let token: Promise<string> | null;
+  const refresh = (): void => {
+    /* eslint-disable @typescript-eslint/camelcase */
+    const url = `${squidex.baseUrl}/identity-server/connect/token`;
+    const res = Got.post(url, {
+      form: {
+        grant_type: 'client_credentials',
+        scope: 'squidex-api',
+        client_id: squidex.clientId,
+        client_secret: squidex.clientSecret,
+      },
+    }).json();
+
+    token = res.then((r) => (r as { access_token: string }).access_token);
+    /* eslint-enable @typescript-eslint/camelcase */
```

Comment:
```suggestion
    /* eslint-enable @typescript-eslint/camelcase */
    token = res.then((r) => (r as { access_token: string }).access_token);
```

Reply:
I need the disable on the promise on line 23.

Reply:
You should disable the rule for the line after. Not the line before